framework/source/class/qx/test/Bootstrap.js

```diff
@@ -292,7 +292,7 @@ qx.Class.define("qx.test.Bootstrap",
 
           stopEngine : function()
           {
-            var ret = arguments.callee.base.call();
+            var ret = qx.test.Bmw.prototype.stopEngine.base.call();
```

Comment:
yes, really old - I guess this is why it was doing some slightly strange things to accomplish the tests; I was aiming to make the smallest changes and just translate the syntax, but I agree it doesn't make it any clearer!  I'll change it